grafana-plugin/src/containers/OutgoingWebhook2Form/OutgoingWebhook2Form.tsx

```diff
@@ -19,28 +24,46 @@ const cx = cn.bind(styles);
 
 interface OutgoingWebhook2FormProps {
   id: OutgoingWebhook2['id'] | 'new';
-  action: 'new' | 'update';
+  action: WebhookFormActionType;
   onHide: () => void;
   onUpdate: () => void;
+  onDelete: () => void;
 }
 
+export const WebhookTabs = {
+  Settings: new KeyValuePair('Settings', 'Settings'),
+  LastRun: new KeyValuePair('LastRun', 'Last Run'),
+};
+
 const OutgoingWebhook2Form = observer((props: OutgoingWebhook2FormProps) => {
-  const { id, action, onUpdate, onHide } = props;
+  const history = useHistory();
+  const { id, action, onUpdate, onHide, onDelete } = props;
+  const [activeTab, setActiveTab] = useState<string>(
+    action === WebhookFormActionType.EDIT_SETTINGS ? WebhookTabs.Settings.key : WebhookTabs.LastRun.key
+  );
 
   const store = useStore();
 
   const { outgoingWebhook2Store } = store;
+  const isNewOrCopy = action === WebhookFormActionType.NEW || action === WebhookFormActionType.COPY;
+
+  if (
+    (action === WebhookFormActionType.EDIT_SETTINGS || action === WebhookFormActionType.VIEW_LAST_RUN) &&
+    !outgoingWebhook2Store.items[id]
+  ) {
+    return null;
+  }
 
   const data =
     id === 'new'
```

Comment:
it's confusing a bit, what the difference between `action === WebhookFormActionType.NEW` and `id === 'new'`

Reply:
Agree, I'll refactor it